themes/powerline/powerline.base.bash

```diff
@@ -13,7 +13,7 @@ function __powerline_user_info_prompt {
   local user_info=""
   local color=${USER_INFO_THEME_PROMPT_COLOR}
 
-  if ${CAN_SUDO}; then
+  if [[ "THEME_CHECK_SUDO" = true ]]; then
```

Comment:
Sorry, my bad - please add a `$` in here for the start of the variable. It should read 

```bash
if [[ "$THEME_CHECK_SUDO" = true ]]; then
```

Looks like I missed the dollar sign in my last comment, sorry about that.